src/gwt/src/org/rstudio/studio/client/workbench/views/source/Source.java

```diff
@@ -627,7 +495,16 @@ else if (userPrefs_.editorKeybindings().getValue() == UserPrefs.EDITOR_KEYBINDIN
          ShortcutManager.INSTANCE.setEditorMode(KeyboardShortcut.MODE_SUBLIME);
       else
          ShortcutManager.INSTANCE.setEditorMode(KeyboardShortcut.MODE_DEFAULT);
-   
+
+      initialized_ = true;
+
+      // !!! comment why this is needed
```

Comment:
Might as well comment it!!!

Reply:
If I remembered :( I did some testing without the event and didn't notice anything breaking. I will continue to test but for now I've removed it.